template.yaml

```diff
@@ -1019,7 +1019,6 @@ Resources:
       CodeUri: scopus-import
       Handler: no.sikt.nva.scopus.ScopusHandler::handleRequest
       Role: !GetAtt LambdaRole.Arn
-      ReservedConcurrentExecutions: 50
```

Comment:
Remember that scopes-import will always run in 4 environment in parallell. All 4 environments will receive the same email at the same time. This might stress the channel-registry.